configs/app.dev.json

```diff
@@ -0,0 +1,19 @@
+{
+    "type": "gpii.appWithTaskTray",
```

Comment:
Please distinguish the type name in each config file to be more specific with the goal of this config.